libwayshot/src/lib.rs

```diff
@@ -572,8 +577,8 @@ impl WayshotConnection {
                             image_util::rotate_image_buffer(
                                 image,
                                 frame_copy.transform,
-                                frame_copy.frame_format.size.width,
-                                frame_copy.frame_format.size.height,
+                                frame_copy.logical_region.inner.size,
```

Comment:
Emm. do it need to resize here? I think use the size of the image will be fine, because before, the resize is finally done in the finally output

Reply:
I mean before the logic is shoting all picture, and finally resize them to the size of the selected region, and convert them together. so before the picture do not need to do anything, finally they will all be the size of the selected area, so I cannot understand now the logic is. because after it will be resized again